pyo3-macros-backend/src/introspection.rs

```diff
@@ -170,18 +152,45 @@ pub fn function_introspection_code(
     IntrospectionNode::Map(desc).emit(pyo3_crate_path)
 }
 
-fn const_introspection_code<'a>(ident: &'a Ident, value: &'a String) -> IntrospectionNode<'a> {
-    IntrospectionNode::Map(
-        [
-            ("type", IntrospectionNode::String("const".into())),
-            (
-                "name",
-                IntrospectionNode::String(ident.unraw().to_string().into()),
-            ),
-            ("value", IntrospectionNode::String(value.into())),
-        ]
-        .into(),
-    )
+pub fn attribute_introspection_code(
+    pyo3_crate_path: &PyO3CratePath,
+    parent: Option<&Type>,
+    name: String,
+    value: String,
+    mut rust_type: Type,
+    r#final: bool,
```

Comment:
rather than `r#final`, should we go for something like `is_final`?

Reply:
Indeed, much better. Done!